### ISSUE TITLE
página de vagas python

### ISSUE THREAD

**Issue by humrochagf** [pythonbrasil/wiki#37]
uma página para divulgar as vagas de python no brasil

**Comment by luzfcb** [comment]
Uma duvida, alguma ideia de como saber se a vaga X ainda existe, para podermos remove-la caso não exista?

**Comment by diegorocha** [comment]
Não consigo pensar numa forma automática. A pessoa que postou teria que notificar pra tirar a vaga depois, ou a gente tirar automaticamente depois de x tempo (caso ainda esteja aberta a pessoa pede pra incluir de novo).

**Comment by diegorocha** [comment]
Mas a estrutura pra postar as vagas pode ser a mesma estrutura os grupos locais.
Seria possível postar uma vaga sem ter que editar o .md só criando um json.

**Comment by erichideki** [comment]
Por mim deixava a vaga postada para a posteridade e documentação. Simples assim, é uma vaga que abre no momento em que é postada, cabe ao candidato entrar em contato. 

Muitas vezes a pessoa encaminha CV a empresa, e de repente ele se torna um candidato viável, mesmo que a empresa não tenha vagas abertas no momento.

**Comment by luzfcb** [comment]
Fechou @erichideki . sem mais perguntas. deixa a vaga publicada contanto que tenha data de publicação.

**Comment by diegorocha** [comment]
@erichideki @luzfcb justo. Quem vai mandar o CV vai estar ciente

**Comment by humrochagf** [comment]
o que vcs acham de usar a estrutura de blog fo pelican pras vagas e trocar a estrutura de eventos pro mesmo esquema de grupys?

pq pelo que eu vi é mais prático o esquema de blog pra vaga que o cara só joga o texto de descrição da vaga num md e com a data de publicação da vaga e tags pras áreas de conhecimento

e evento como tem um formato mais estruturado usa a estrutura de json

**Comment by erichideki** [comment]
@humrochagf Concordo com sua sugestão. Por mim está ok. Ter uma lista no formato dos grupys para eventos fica melhor, até porque estão sendo vários :smile: 

Mas o modelo da Pyladies também é muito bom: http://staging-brasil-pyladies.herokuapp.com/events/

Fica as duas sugestões para a página de eventos.

**Comment by humrochagf** [comment]
Sim o layout vai ser diferente,  a estrutura que eu falo é a de organização em arquivos .json com os dados estruturados

**Comment by ferfabricio** [comment]
E se colocasse um tempo de expiração para vaga? Normalmente quem está com uma vaga aberta sabe qual a data limite para contratação de um profissional.

**Comment by erichideki** [comment]
@ferfabricio Entendo seu ponto, porém o ponto é que seja documentado as vagas. Expirá-las se torna algo ruim de controlar, até porque quem dita isso é a empresa que está contratando. 

O importante é listar a vaga, e deixá-la para posteridade como documentação do que a empresa utiliza e etc.

**Comment by humrochagf** [comment]
pode ser uma das informações da vaga @ferfabricio, mas independente de expirar ou não podemos fazer como o @erichideki disse e manter como histórico, a não ser que a empresa em questão se manifeste pedindo a retirada da vaga

**Comment by filipecifali** [comment]
@humrochagf ela mesma pode retirar a vaga se quiser, não precisa de data a expirar a vaga.

**Comment by marianaremedios** [comment]
acho o modelo do @frontendbr bom. Poderiamos copiar...
https://github.com/frontendbr/vagas

**Comment by pixies** [comment]
Quem vai anunciar a vaga vai ter que ter uma conta aqui no github?
Acho a ideia da validade/expirar muito legal. E não precisa(deve) excluir mesmo, mais sim classificar, um status (ativo/inativo). 
Na boa, acho que se preocupar com a qualidade do anuncio da vaga seria massa!

**Comment by adorilson** [comment]
Olá, pessoal.
Isso não foi implementado, né?
Por que não apenas colocar um link para o [PyJobs](https://www.pyjobs.com.br/) no menu Impressione-se?
